### PR TITLE
feat(google): this implements google abstraction

### PR DESCRIPTION
**Pull request by nutrina**
This fixes: #621 

In addition it also contains a fix in the database client + related tests

### CODE REVIEW

File: app/components/CardListV2.tsx [hunk 1]
```diff
@@ -145,6 +146,14 @@ providers.set("GtcStaking", {
   platFormGroupSpec: GtcStaking.GTCStakingProviderConfig,
 });
 
+providers.set("Google", {
+  platform: new Google.GooglePlatform({
+    clientId: process.env.NEXT_PUBLIC_PASSPORT_GOOGLE_CLIENT_ID,
+    redirectUri: process.env.NEXT_PUBLIC_PASSPORT_LINKEDIN_CALLBACK,
```
Comment (schultztimothy) [inline app/components/CardListV2.tsx:152]: Looks like this should be google

File: platforms/src/Google/App-Bindings.ts [hunk 2]
```diff
@@ -0,0 +1,28 @@
+/* eslint-disable */
```
Comment (schultztimothy) [inline platforms/src/Google/App-Bindings.ts:1]: You should be able to remove this

**Review by schultztimothy — APPROVED** [review]
Two tiny comments, but LGTM 🚀

**Comment by farque65** [comment]
I am seeing the following issue when testing locally
Screen Shot 2022-11-17 at 3 54 58 PM

**Comment by nutrina** [comment]
> I am seeing the following issue when testing locally Screen Shot 2022-11-17 at 3 54 58 PM

You need to create a new oauth app, and register the proper redirect URI.
We will need to do this also for the  review, staging and production environments.

File: app/components/CardListV2.tsx (hunk 1 above)
Comment (nutrina) [reply app/components/CardListV2.tsx]: Fixed

**Pull request merged**